packages/components/src/components/Menu/Menu.test.tsx

```diff
@@ -36,4 +36,21 @@ describe("<Menu />", () => {
     expect(screen.getByText("Item 1")).toBeInTheDocument();
     expect(screen.getByText("Item 2")).toBeInTheDocument();
   });
+
+  it("allows a disabled prop for a menu item and disables the button", () => {
+    render(
+      <Menu items={[{ onClick: jest.fn(), label: "Item 1", disabled: true }]} />
+    );
+
+    const menuButton = screen.getByRole("button");
+    expect(menuButton).toBeInTheDocument();
+
+    userEvent.click(menuButton);
+
+    expect(screen.getByRole("presentation")).toBeInTheDocument();
+
+    expect(
+      screen.getByRole("button", { name: /item 1/i, hidden: true })
```

Comment:
Why do you need the hidden true here?

Reply:
I'm not exactly sure @apvale . But the button is hidden in the dom for some reason. Although I can see it in story book and in the UI, when testing, it is hidden.

Reply:
Well I figured out that it is because of the async/await. It works when I removed hidden and I added the async/await with the user click.